tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/CommencementDateControllerTests.cs

```diff
@@ -63,6 +63,53 @@ public async Task Update_NoOrderFound_ReturnsNotFound()
             result.Should().BeOfType<NotFoundResult>();
         }
 
+        [Test]
+        public async Task Get_WithCommencementDate_ReturnsOkResult()
+        {
```

Comment:
Could you make the two Gets with OkResult move into one test and use TestCases, as they are very similar